src/services/chartdata.service.ts

```diff
@@ -0,0 +1,59 @@
+import { Between, FindManyOptions, Repository } from 'typeorm';
```

Comment:
please use camel case to separate words in file names: chartdata.service.ts -> chartData.service.ts

Reply:
I moved the fields into the constants file. then created the fields type

![image](https://user-images.githubusercontent.com/69632363/127425775-65b72b26-db76-43c0-8c67-290b63db4ad3.png)
![image](https://user-images.githubusercontent.com/69632363/127425784-023bc9ba-bdd7-4975-bbec-8d615e0789c3.png)